src/components/KTableView/KTableView.vue

```diff
@@ -407,6 +407,12 @@ enum TableViewHeaderKeys {
   BULK_ACTIONS = 'bulkActions',
 }
 
+interface DataSelectState {
+  rowKey: string
+  selected: boolean
+  disabled: boolean
+}
```

Comment:
nit: this should be moved to the types file

Reply:
This is an internal interface used by internal ref only -- we should not be exporting it (same deal as with `TableViewHeaderKeys` enum).

Reply:
our pattern is to define all types in the `/types` directory; even if it's internal, it doesn't hurt to export it.

Reply:
In that case should the `TableViewHeaderKeys` enum be moved to the `/types` directory too?

Reply:
Done